File: GridView/GridObjectRelationalCRUD/gridobjectrelationalcrud-cs/GridObjectRelationalCRUD-CS/Model/Artist.cs
using System;
using System.Collections.Generic;

namespace GridObjectRelationalCRUD.Models
{
    [Serializable]
    public class Artist
    {
        private int id;
        private string name;
        private List<Album> albums = new List<Album>();

        public Artist()
        {

        }

        public Artist(int id, string name)
        {
            this.id = id;
            this.name = name;
        }

        public int Id
        {
            get { return id; }
            set { id = value; }
        }
        
        public string Name
        {
            get { return name; }
            set { name = value; }
        }

        public List<Album> Albums
        {
            get { return albums; }
        }
    }
}

File: GridView/GridObjectRelationalCRUD/gridobjectrelationalcrud-vb/GridObjectRelationalCRUD-VB/Model/Artist.vb
<Serializable> _
Public Class Artist
    Private m_id As Integer
    Private m_name As String
    Private m_albums As New List(Of Album)()


    Public Sub New()
    End Sub

    Public Sub New(id As Integer, name As String)
        Me.m_id = id
        Me.m_name = name
    End Sub

    Public Property Id() As Integer
        Get
            Return m_id
        End Get
        Set(value As Integer)
            m_id = value
        End Set
    End Property

    Public Property Name() As String
        Get
            Return m_name
        End Get
        Set(value As String)
            m_name = value
        End Set
    End Property

    Public ReadOnly Property Albums() As List(Of Album)
        Get
            Return m_albums
        End Get
    End Property
End Class